support-config/src/main/resources/touchpoint.PROD.conf

```diff
@@ -3,21 +3,23 @@ touchpoint.backend.environments {
     stripe {
       AU {
         api.key.public = "pk_live_57xV50eFjPIA990PEGWJwoHp"
-        api.key.secret = "dummy-key"
       }
       US {
-        api.key.public = "pk_live_FCXf4hYCh6eefkHDbwKvvwZR00AFIyucU3"
-        api.key.secret = "dummy-key"
+        api.key.public = "pk_live_2O6zPMHXNs2AGea4bAmq5R7Z"
```

Comment:
this was wrong, for recurring US we use the default key not the oneoff one.  This was getting ovverriden by the correct values that were in parameter store.
This file is correct now, and we can delete all the duplicated values from parameter store once this is shipped